docs/patterns/ComponentFrameworkCoverage.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Framework Support Matrix
+outline: false
+---
+
+<script setup>
+// No import needed if WarpBadge is globally registered
+
+// 1) Load all component data.json files (relative to this page inside /components)
+const modules = import.meta.glob('./**/data.json', { eager: true })
+
+// 2) Normalize framework names to your canonical headers
+const NAME_MAP = {
```

Comment:
Can you reuse all these from ".vitepress/theme/components/utils/index.js" instead of re-declaring them once more?